src/Invictus.Testing/LogicAppClient.cs

```diff
@@ -0,0 +1,498 @@
+                Status = "Succeeded"
+            };
+
+            return await TemporaryEnableStaticResultAsync(actionName, successfulStaticResult);
+        }
+
+        /// <summary>
+        /// Temporary enables a static result for an action with the given <paramref name="actionName"/> on the logic app,
+        /// and disables the static result when the returned instance gets disposed.
+        /// </summary>
+        /// <param name="actionName">The name of the action to enable the static result.</param>
+        /// <param name="definition">The definition that describes the static result for the action.</param>
+        /// <returns>
+        ///     An instance to control when the static result for the action on the logic app should be disabled.
+        /// </returns>
+        public async Task<IAsyncDisposable> TemporaryEnableStaticResultAsync(string actionName, StaticResultDefinition definition)
+        {
+            Guard.NotNullOrEmpty(actionName, nameof(actionName));
+            Guard.NotNull(definition, nameof(definition));
+
+            return await AsyncDisposable.CreateAsync(
+                async () => await EnableStaticResultForActionsAsync(new Dictionary<string, StaticResultDefinition> { [actionName] = definition }),
+                async () => await DisableStaticResultForActionAsync(actionName));
+        }
+
+        /// <summary>
+        /// Enables static results for a given set of actions on the logic app.
+        /// </summary>
+        /// <param name="actions">The set of action names and the corresponding static result.</param>
+        /// <returns>
+        ///     An instance to control when the static result for the actions on the logic app should be disabled.
+        /// </returns>
+        public async Task<IAsyncDisposable> TemporaryEnableStaticResults(IDictionary<string, StaticResultDefinition> actions)
+        {
+            Guard.NotNull(actions, nameof(actions));
+            Guard.NotAny(actions, nameof(actions));
+
+            return await AsyncDisposable.CreateAsync(
+                async () => await EnableStaticResultForActionsAsync(actions),
+                async () => await DisableStaticResultsForActionsAsync(actions.Keys));
+        }
+
+        /// <summary>
+        /// Enables a static result for an action with the given <paramref name="actionName"/> on the logic app.
+        /// </summary>
+        /// <param name="actionName">The name of the action to enable the static result.</param>
+        /// <param name="staticResultDefinition">The definition that describes the static result for the action.</param>
+        /// <returns>
+        ///     An indication whether the enabling was successful [true] or not [false].
+        /// </returns>
+        public async Task<bool> EnableStaticResultForActionAsync(string actionName, StaticResultDefinition staticResultDefinition = null)
```

Comment:
Can we add a test that uses this?

Reply:
Hmm, I was actually planning to make this `private`.

Reply:
Made private.